package.json

```diff
@@ -43,7 +43,7 @@
         "docs:backend": "cd backend && documentation build index.js -c documentation.yml -f md > README.md",
         "generate-type-declarations": "tsc -p src --checkJs false && tsc -p backend --checkJs false",
         "lint": "npm run lint:code",
-        "lint:code": "eslint --cache --ignore-path .gitignore --max-warnings 10 .",
+        "lint:code": "eslint --cache --ignore-path .gitignore .",
```

Comment:
There are no warnings ATM and I don't see why we would allow them.  `--max-warnings` was removed from stage a while back, so I thought it's worth doing that here too

Reply:
Definitely, good idea to remove it.